service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentControllerTest.java

```diff
@@ -0,0 +1,122 @@
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.EMAIL;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocument;
+
+@ActiveProfiles("integration-test")
+@WebMvcTest(SendDocumentController.class)
+@OverrideAutoConfiguration(enabled = true)
+class SendDocumentControllerTest extends AbstractControllerTest {
+
+    private static final String SERVICE_AUTH_TOKEN = "Bearer service token";
+    private static final byte[] PDF = {1, 2, 3, 4, 5};
+
+    @MockBean
+    private DocmosisCoverDocumentsService docmosisCoverDocumentsService;
+
+    @MockBean
+    private DocumentDownloadService documentDownloadService;
+
+    @MockBean
+    private LDClient ldClient;
+
+    @MockBean
+    private SendLetterApi sendLetterApi;
+
+    @MockBean
+    private AuthTokenGenerator authTokenGenerator;
+
+    SendDocumentControllerTest() {
+        super("send-document");
+    }
+
+    @BeforeEach
+    void setup() {
+        given(ldClient.boolVariation(anyString(), any(), anyBoolean())).willReturn(true);
+        given(documentDownloadService.downloadDocument(anyString())).willReturn(PDF);
+        given(docmosisCoverDocumentsService.createCoverDocuments(anyString(), anyLong(), any())).willReturn(
+            DocmosisDocument.builder().bytes(PDF).build());
+        given(authTokenGenerator.generate()).willReturn(SERVICE_AUTH_TOKEN);
+    }
+
+    @Test
+    void shouldSendDocumentToRepresentativesWithPostServingPreferences() {
+        Representative representative1 = representative("John Smith", POST);
+        Representative representative2 = representative("Alex Brown", EMAIL);
+        Representative representative3 = representative("Emma White", DIGITAL_SERVICE);
+
+        DocumentReference documentToBeSend = testDocument();
+
+        CaseDetails caseDetails = buildCaseData(documentToBeSend, representative1, representative2, representative3);
+
+        postAboutToSubmitEvent(caseDetails);
+
+        verify(documentDownloadService).downloadDocument(documentToBeSend.getBinaryUrl());
+        verify(sendLetterApi).sendLetter(anyString(), any(LetterWithPdfsRequest.class));
+    }
+
+    @Test
+    void shouldNotSendDocumentWhenFeatureToggleIsOff() {
+        given(ldClient.boolVariation(anyString(), any(), anyBoolean())).willReturn(false);
+        DocumentReference documentToBeSend = testDocument();
+        CaseDetails caseDetails = buildCaseData(documentToBeSend);
+
+        postAboutToSubmitEvent(caseDetails);
+
+        verify(documentDownloadService, never()).downloadDocument(documentToBeSend.getBinaryUrl());
```

Comment:
verify(documentDownloadService, never()).downloadDocument(any());
It ensure that nothing is downloaded